roles/chameleon_mariadb/tasks/site.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I think a better name for this task is `config`, as it is configuring the backup script on the MariaDB host.